src/gui_common/CustomDialog.cs

```diff
@@ -436,31 +436,32 @@ private void HandleActiveDrag()
         else
         {
             // Handle border dragging
+            var screenSize = (GetViewport()?.GetVisibleRect().Size).GetValueOrDefault();
```

Comment:
I think we should print an error an exit this method if the viewport doesn't exist. I'm pretty sure I've done pretty hard errors in the past about missing critical objects. The viewport can only be missing if this isn't attached to the scene tree so this wouldn't be able to receive the _GuiInput event anyway in that case.

Reply:
I just copied this from the function below. So maybe we can just remove it.

Reply:
Probably. Might also want to move this to a helper method as this is pretty dense logic that doesn't make much sense if it is edited in one place and not the other (even if that happens accidentally).